AlphaWallet/KeyManagement/EtherKeystore.swift

```diff
@@ -127,12 +127,19 @@ open class EtherKeystore: NSObject, Keystore {
 
     func createAccount(completion: @escaping (Result<AlphaWallet.Address, KeystoreError>) -> Void) {
         DispatchQueue.global(qos: .userInitiated).async { [weak self] in
-            guard let strongSelf = self else {
-                return
-            }
-            let result = strongSelf.createAccount()
-            OperationQueue.main.addOperation {
-                completion(result)
+            guard let strongSelf = self else { return }
+
+            let mnemonicString = strongSelf.generateMnemonic()
```

Comment:
for now when we sink for walletsPublisher, it receives events not only on main queue, as function
`func createAccount(completion: @escaping (Result<AlphaWallet.Address, KeystoreError>) -> Void)` generates mnemonic and updates wallet on background. It would be easier when we update `wallets` on main queue, as well as receive events on main queue. This PR generates mnemonic on `Background Queue` and updates wallets on main queue

Reply:
It does the same, but generates mnemonic on `background queue`, and makes import on `main`

Reply:
OK